packages/botonic-react/src/components/index.d.ts

```diff
@@ -50,8 +54,13 @@ export interface ButtonProps {
 
 export const Button: React.FunctionComponent<ButtonProps>
 
+export interface ReplyProps {
+  path?: string
+  payload?: string
+  children: string
+}
 export const Reply: React.FunctionComponent<ReplyProps>
-export type ReplyProps = ButtonProps
+export type Reply = React.FunctionComponent<MessageProps>
```

Comment:
Is this correct or it should be `React.FunctionComponent<ReplyProps>`?

Reply:
yes, changed!